src/theory/quantifiers/sygus/sygus_grammar_cons.cpp

```diff
@@ -2,7 +2,7 @@
 /*! \file sygus_grammar_cons.cpp
  ** \verbatim
  ** Top contributors (to current version):
- **   Andrew Reynolds, Haniel Barbosa, Mathias Preiner
+ **   Andrew Reynolds, Haniel Barbosa, Aina Niemetz
```

Comment:
Fight! :smile:

Reply:
Haha, no fights, I just called the update-copyright script on it :D